### PR TITLE
[15.0][MIG] stock_picking_backorder_strategy: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT37321

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

**Review by rousseldenis — COMMENTED** [review]
@ernesto-garcia-tecnativa Thanks for this. It seems that several tests are not evaluated (see coverage).

Could you check ?

File: stock_picking_backorder_strategy/tests/test_backorder_strategy.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    def test_backorder_strategy_create(self):
+        """Check strategy for stock.picking_type_in is create
+        Receive picking
+        Check backorder is created
+        """
+        self.picking_type.backorder_strategy = "create"
+        self._process_picking()
+        backorder = self.picking_obj.search([("backorder_id", "=", self.picking.id)])
+        if len(backorder):
```
Comment (rousseldenis) [inline stock_picking_backorder_strategy/tests/test_backorder_strategy.py]: This should be removed as assert should always be evaluated

**Review by pedrobaeza — APPROVED** [review]
Code and functional review

File: stock_picking_backorder_strategy/tests/test_backorder_strategy.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Copyright 2018 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests import common
+from odoo.tests.common import TransactionCase
```
Comment (chienandalu) [inline stock_picking_backorder_strategy/tests/test_backorder_strategy.py]: Absolutly non blocking: you've got `TransactionCase` in `odoo.tests` namespace as well.

File: stock_picking_backorder_strategy/tests/test_backorder_strategy.py (hunk 1 above)
Comment (chienandalu) [reply stock_picking_backorder_strategy/tests/test_backorder_strategy.py]: Sure :woozy_face:  I see it slipped in the v11 migration... More than remove the test should be redesigned to fulfill its purpose.

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza  @chienandalu I made some changes to tests, review please

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@chienandalu Could you update your review ?

**Review by chienandalu — APPROVED** [review]
Thanks @ernesto-garcia-tecnativa :+1: !

**Pull request merged**